config/routes.rb

```diff
@@ -1,58 +1,11 @@
 Rails.application.routes.draw do
   root 'homes#index'
   devise_for :users
-  # The priority is based upon order of creation: first created -> highest priority.
-  # See how all your routes lay out with "rake routes".
 
-  # You can have the root of your site routed with "root"
-  # root 'welcome#index'
+  devise_scope :user do
+    get  'sitters/sign_up' => 'devise/registrations#new_sitter', :as => 'new_sitter_registration'
```

Comment:
Line is too long. [97/80]<br>Unnecessary spacing detected.<br>Put one space between the method name and the first argument.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.